frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/disk-wrapper.ts

```diff
@@ -3,53 +3,33 @@ import { V1Disk } from '../../../types/vm/disk/V1Disk';
 import { DiskType, DiskBus } from '../../../constants/vm/storage';
 
 type CombinedTypeData = {
-  bus?: DiskBus;
+  bus?: string;
```

Comment:
Just curious, but why did you drop to a less specific type here?

Reply:
because we use the raw data in the internal structure of the Wrapper which is string. The handling of the CombinedTypeData and sanitize function were moved to the baseClass. The DiskWrapper can still expose the bus as DiskBus enum in the getter